spec/unit/lib/cloud_controller/bits_expiration_spec.rb

```diff
@@ -41,19 +41,47 @@ module VCAP::CloudController
       end
     end
 
+    context 'with docker cf apps' do
+      before do
+        t        = Time.now
+        @current = DropletModel.make(
+          app_guid:     app.guid,
+          created_at:   t,
+          droplet_hash: nil,
+          docker_receipt_image: 'repo/test-app',
+        )
+        app.update(droplet: @current)
+
+        10.times do |i|
+          DropletModel.make(
+            app_guid:     app.guid,
+            created_at:   t + i,
+            droplet_hash: nil,
+            docker_receipt_image: 'repo/test-app',
+          )
+        end
+      end
+
+      it 'it does not enqueue a job to delete the blob' do
+        expect { BitsExpiration.new.expire_droplets!(app) }.not_to change { Delayed::Job.count }.from(0)
```

Comment:
`not_to change` does not require the `from(0)`

Reply:
Ok